internal/handler/http.go

```diff
@@ -196,11 +197,11 @@ func ProvideHandler() fx.Option {
 		},
 	)
 }
-func New(sw *SinkWrapper, log *zap.Logger, t *HandlerTelemetry, maxOutstanding, incomingQueueDepth int64) (*ServerHandler, error) {
+func New(sw *sink.SinkWrapper, log *zap.Logger, t *HandlerTelemetry, maxOutstanding, incomingQueueDepth int64) (*ServerHandler, error) {
```

Comment:
a general note, let's be consistent with names such as `log *zap.Logger` vs `logger *zap.Logger` if we aren't already